internal/display/statusline.go

```diff
@@ -47,6 +47,12 @@ var statusInfo = map[string]func(*buffer.Buffer) string{
 		}
 		return ""
 	},
+	"overwrite": func(b *buffer.Buffer) string {
```

Comment:
Maybe `overwritemode` is more precise, as suggested in the linked feature request.

Reply:
I think I'd prefer `overwrite`.